mappings/net/minecraft/world/gen/feature/util/CaveSurface.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_5721 net/minecraft/world/gen/feature/util/DripstoneColumn
+CLASS net/minecraft/class_5721 net/minecraft/world/gen/feature/util/CaveSurface
```

Comment:
Hmm, interesting

Reply:
This class is used for finding the ceiling or floor of a cave. Before 21w10a, this was only used by dripstone, so that's where the name is coming from. In 21w10a, the `cave_surface` decorator was added, which also uses this.